src/components/app-content-renderer/recommendation-tile.js

```diff
@@ -21,28 +13,28 @@ import {
 import { useIntl } from 'react-intl';
 
 import useRequest from './use-request';
-
-const groupIconMapper = {
-  automation: ProcessAutomationIcon,
-  exclamationTriangle: ExclamationTriangleIcon,
-  exclamationCircle: ExclamationCircleIcon,
-  infoCircle: InfoCircleIcon,
-  checkCircle: CheckCircleIcon,
-  security: SecurityIcon,
-  unknown: QuestionCircleIcon,
-  default: QuestionCircleIcon,
-};
+import iconMapper from '../../utils/icon-mapper';
+import { useDispatch } from 'react-redux';
+import { removeRecommendationTile } from '../../store/actions';
 
 const RecommendationGroup = (recommendation) => {
   const intl = useIntl();
-  const [{ count, response, show }] = useRequest(recommendation);
+  const dispatch = useDispatch();
+  const removeTile = ({ show }) =>
+    show === false &&
+    dispatch(removeRecommendationTile(recommendation.id, recommendation.recId));
+  const [{ count, response, show }] = useRequest(
+    recommendation,
+    removeTile,
+    () => removeTile({ show: false })
```

Comment:
should be true I think

Reply:
No, it should be false, we want to remove tiles on API errors.

Reply:
oh, sorry! 🤦